src/main/resources/static/views/admin/AdminPanelPage.html

```diff
@@ -0,0 +1,284 @@
+                                <a href="#" ><span class="glyphicon glyphicon-trash" title="Trash"></span></a></td>
+                        </tr>
+                        <tr>
+                            <td><input type="checkbox"></td>
+                            <td>Dolor Lorem Amet</td>
+                            <td>Alan Johnson</td>
+                            <td>26th May 2017</td>
+                            <td><a href="#" ><span class="glyphicon glyphicon-pencil" title="Edit"></span></a>
+                                <a href="#" ><span class="glyphicon glyphicon-trash" title="Trash"></span></a></td>
+                        </tr>
+                        <tr>
+                            <td><input type="checkbox"></td>
+                            <td>Dolor Lorem Amet</td>
+                            <td>Dobby</td>
+                            <td>26th May 2017</td>
+                            <td><a href="#" ><span class="glyphicon glyphicon-pencil" title="Edit"></span></a>
+                                <a href="#" ><span class="glyphicon glyphicon-trash" title="Trash"></span></a></td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </article>
+
+        <article class="module width_quarter">
+            <header><h3>Messages</h3></header>
+            <div class="message_list">
+                <div class="module_content">
+                    <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                        <p><strong>Max</strong></p></div>
+                    <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                        <p><strong>Max</strong></p></div>
+                    <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                        <p><strong>Max</strong></p></div>
+                    <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                        <p><strong>Max</strong></p></div>
+                    <div class="message"><p>Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor.</p>
+                        <p><strong>Max</strong></p></div>
+                </div>
+            </div>
+            <footer>
+                <form class="post_message">
+                    <input type="text" value="Message" onfocus="if(!this._haschanged){this.value=''};this._haschanged=true;">
+                    <input type="submit" class="btn_post_message" value=""/>
+                </form>
+            </footer>
+        </article>
+
+        <div class="clear"></div>
+
+        <article class="module width_full">
+            <header><h3>Post New Article</h3></header>
+            <div class="module_content">
+                <fieldset>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a <legend> tag to this fieldset. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AFieldsetWithoutLegendCheck)